src/main/scala/firrtl/stage/transforms/UpdateAnnotations.scala

```diff
@@ -22,8 +22,8 @@ class UpdateAnnotations(val underlying: Transform) extends Transform with Wrappe
 
     logger.info(s"Form: ${result.form}")
 
```

Comment:
Seems like you might want to remove this blank line to make it exactly like the code in Compiler.scala. IntelliJ is catching more and more duplicated code, the more the same the more easily matched.
```suggestion
```